system/templates/rolls/roll-graphic.hbs

```diff
@@ -0,0 +1,54 @@
+<div class="dice-roll ironsworn-roll">
+
+  <div class="dice-formula">
+    <p>
```

Comment:
i wouldn't consider this a paragraph (on account of it not being prose), so `<p>` wouldn't be my first choice. and as a styling wrapper it's redundant with its parent `<div>`

Reply:
Will address with #366.